packages/zapier/src/types/measurement.ts

```diff
@@ -0,0 +1,8 @@
+export type MeasurementInputData = {
+  hardwareId: string;
+  testId: string;
+  name: string;
+  type: "boolean" | "dataframe";
```

Comment:
I would extract this out into a type `MeasurementType`, you can export it too so that you don't have to do `MeasurementInputData["type"]`